src/sqlfluff/cli/commands.py

```diff
@@ -696,6 +696,11 @@ def do_fixes(lnt, result, formatter=None, **kwargs):
         "or in the .sqlfluff config file."
     ),
 )
+@click.option(
+    "--show_lint_violations",
```

Comment:
Most of our command lines use dashes rather than underscores (`--disable_progress_bar` is the exception and we should probably change that):

```suggestion
    "--show-lint-violations",
```